packages/sitecore-jss-react/src/components/EditingScripts.tsx

```diff
@@ -15,20 +16,24 @@ export const EditingScripts = (): JSX.Element => {
   if (pageState === LayoutServicePageState.Normal) return <></>;
 
   if (editMode === EditMode.Metadata) {
+    let jssClientData: Record<string, unknown> = clientData;
+    if (pageState === LayoutServicePageState.Edit) {
```

Comment:
I assume that this check is extra, isn't it? since We are already in "metadata" mode